_pages/get_involved.md

```diff
@@ -70,9 +70,10 @@ can be found in the BIDS [specification](https://bids-specification.readthedocs.
 
 <br>
 
-Some proposals that set out to extend the BIDS specification have instead become tools for handling BIDS.
+Some proposals that set out to extend the BIDS specification have instead become tools for handling BIDS or have been merged into other BEPs.
 See the table below.
 
-| Extension label                                                 | Title        | Moderators/leads                               | Tool name                                                                 |
-| --------------------------------------------------------------- | ------------ | ---------------------------------------------- | ------------------------------------------------------------------------- |
-| [BEP015](https://bids.neuroimaging.io/bep015){:target:"_blank"} | Mapping file | Eric Earl, Camille Maumet, and Vasudev Raguram | [File mapper](https://github.com/DCAN-Labs/file-mapper){:target:"_blank"} |
+| Extension label                                                 | Title                          | Moderators/leads                               | Outcome                                                                   |
+| --------------------------------------------------------------- | ------------------------------ | ---------------------------------------------- | ------------------------------------------------------------------------- |
+| [BEP012](https://bids.neuroimaging.io/bep012){:target:"_blank"} | Resting state fMRI derivatives | Steven Giavasis                                | Merged into BEP012                                                        |
```

Comment:
shouldn't it say BEP013 here?

Reply:
oh god... I have the attention span of an oyster!!!

Reply:
should be fixed
thanks for catching this